opensrp-unicef-angola/build.gradle

```diff
@@ -207,7 +207,7 @@ dependencies {
 
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
 
-    implementation('org.smartregister:opensrp-client-child:0.7.1-VLOGS-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-child:0.7.3-SNAPSHOT@aar') {
```

Comment:
I think we need to publish the **Verbose logging** snapshot for `0.7.3`

Reply:
`0.7.1-VLOGS-SNAPSHOT` was not working on my end